pkg/backends/utils.go

```diff
@@ -76,8 +76,10 @@ func listenerSession(ctx context.Context, lf listenFunc, af acceptFunc, lcf list
 		return nil, err
 	}
 	sessChan := make(chan netceptor.BackendSession)
+	netceptor.MainInstance.BackendAdd()
```

Comment:
these calls to MainInstance are problematic in the case where netceptor is imported. I will look into this